### PR TITLE
Get real summary statistics

### LINKED ISSUE #45: Get summary statistics from the backend
A backend that supplies the necessary ensemble statistics is being developed. The API is settled, so we can now code to it. Do this and use it to drive the Summary tab.

### PR DESCRIPTION
**Pull request by rod-glover**
This PR:
- Fetches real summary stats from the backed
- Extends summary tab configuration to include absolute vs relative values, value precision (sig figs)
- Handles backend fetch errors and shows "n/a/" for missing data
- Displays stats for all data variables (configuration update)

Resolves #45

### CODE REVIEW

File: src/data-services/summary-stats.js [hunk 1]
```diff
@@ -21,18 +21,20 @@ export const fetchSummaryStatistics = (
...
   // TODO: Remove fakery when backend available
   if (process.env.REACT_APP_SUMMARY_STATISTICS_URL === 'fake') {
-    return Promise.resolve(fake[variable]);
+    return new Promise(resolve => {
+      setTimeout(() => resolve(fake[variable]), 1000);
+    });
   }
   return axios.get(
-    urljoin(process.env.REACT_APP_SUMMARY_STATISTICS_URL, 'tbd'),
+    urljoin(process.env.REACT_APP_SUMMARY_STATISTICS_URL, 'percentileanomaly'),
```
Comment (jameshiebert) [inline src/data-services/summary-stats.js]: Given that we've already run in to error situations with the percentiles backend, I'd propose some handling of error conditions (400s, 500s) here or where ever is appropriate.

Comment (rod-glover) [reply src/data-services/summary-stats.js]: Done

**Comment by rod-glover** [comment]
Having received LGTM's on Zulip, merging.

**Comment by tqmurdock** [comment]
What is the order of operations for how the median and percentiles are computed for regions? 

For example for the median taken at every grid box first and then those averaged together?

Or is the regional average taken for each run and then the median of those taken?

**Comment by corviday** [comment]
> What is the order of operations for how the median and percentiles are computed for regions?
> 
> For example for the median taken at every grid box first and then those averaged together?
> 
> Or is the regional average taken for each run and then the median of those taken?
> 

The latter. First the median for each region and model is taken (So there a single value for Vancouver Island CanESM, Vancouver Island ACCESS1-0, etc), and then the twelve values from the PCIC12 ensemble models for the region are used to construct the percentiles (Vancouver Island 75th percentile, etc). 

(The other way happens to be much easier for us, so if that's the preferred calculation, please let us know!)

**Pull request merged**